FrontEnd/FrontEnd.Web/Components/Pages/Recipes/Form.razor

```diff
@@ -91,20 +91,31 @@
         <FluentGridItem xs="12" sm="12" md="6">
             <FluentStack Orientation="Orientation.Vertical">
                 <FluentLabel>Picture</FluentLabel>
-                <!--suppress CssUnresolvedCustomProperty -->
-                <FluentInputFile Id="picture"
-                                 Accept="image/*"
-                                 Style="height: 300px; width: 100%; border: 1px dashed var(--accent-fill-rest);">
-                    <ChildContent>
-                        <label for="picture">
-                            <FluentIcon Value="@(new @Icons.Regular.Size24.ArrowUpload())"/>
-                        </label>
-
-                        <div>
-                            Drag files here you wish to upload, or <label for="picture">browse</label> for them.
-                        </div>
-                    </ChildContent>
-                </FluentInputFile>
+                @if (FormData.Image != null)
+                {
+                    <img src="@FormData.Image" alt="Input image" style="width: 100%"/>
+                    <FluentButton OnClick="() => FormData.Image = null">Remove</FluentButton>
```

Comment:
Perhaps it would look more consistent and minimalistic if we used an icon for removal instead of the word, for example next to the label. We could use disable and keep it there at all times.

<img width="592" alt="Screenshot 2024-05-19 at 11 43 00" src="https://github.com/BenasB/vu-psk/assets/86420873/14eec2cc-ee17-448e-810b-37765fcc0395">

Reply:
But that's just a nitpick I guess, this works.

Reply:
Added a button like this:
![image](https://github.com/BenasB/vu-psk/assets/29711974/9c77fba2-cb23-49f8-8365-a47fa35fd063)